src/components/TextField/OfficeTextField.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <div :style="textFieldStyle" class="ms-TextField">
+        <div class="ms-TextField-wrapper">
+            <OfficeLabel v-if="!!label">
```

Comment:
I really dislike the "bang bang" operation! Just check for `label !== undefined`, as each other value was set by the user himself and might have a purpose.